docs/_drafts/scv2-hodometer.md

```diff
@@ -0,0 +1,68 @@
+Out of the box, it had support for pushing metrics to Google's BigQuery, HTTP endpoints accepting JSON documents, and the local file handle `STDOUT`.
+This made it easy enough to adopt for Core v1, where it's been powering the metrics ever since on monthly active nodes and so on published by Seldon.
+For details on enabling Spartakus for Core v1, please refer to the [official documentation](https://docs.seldon.io/projects/seldon-core/en/latest/workflow/usage-reporting.html).
+
+While convenient in the sense that it had already been written and made collecting _some_ metrics straightforward, Spartakus wasn't an ideal tool for application usage monitoring.
+For a start, the project was deprecated and archived in 2019.
+This led to the creation of the open-source [i-am-spartakus](https://github.com/SeldonIO/i-am-spartakus) repository a few weeks later by Seldon, to retain the project in maintenance mode.
+Plenty of projects are archived, however, and maintaining a fork of something stable isn't much of a burden.
+The larger problem with Spartakus was that the metrics it collected made sense for k8s, but not necessarily for other applications.
+For example, the number of nodes in a cluster has no relation to how many ML deployments or models are on it, or what hardware resources those are consuming.
+It also doesn't provide any information on which version of Core has been installed, or anything else that's application-specific.
+Spartakus does support some custom data in the form of [extensions](https://github.com/kubernetes-retired/spartakus/tree/master/docs#extensions), but this is static information.
+In any case, users may not even want to share all the data on nodes in a cluster that Spartakus collects by default.
+To put it concisely, Spartakus collected lots of data that _wasn't_ of interest but didn't collect lots of data that _was_ of interest!
+
+## Designing Hodometer
+
+Core v2 is a very different kettle of fish from Core v1, and the sorts of metrics we'd want are no exception.
+Core v2 has multiple Kubernetes custom resources --- models, servers, pipelines, and experiments --- compared to just single one in Core v1 --- deployments.
+It furthermore introduces fundamentally new features in the form of multi-model serving (MMS) and over-commit.
+When you consider these things, it should be clear that the metrics for Core v2 will want to be correspondingly more granular and domain-focused.
+
+While Spartakus didn't provide ideal metrics even for Core v1, it did make a number of sensible design decisions to use as a basis for implementing usage metrics in Core v2.
+Specifically, these were:
+* Anonymity via randomised, non-persistent identifiers.
+* Explicitly no collection of PII.
+* Non-integral component.
+* Low periodicity reporting of metrics.
+* Push-based reporting of metrics.
+* Flexibility to configure additional metrics receivers.
+
```

Comment:
:thought_balloon: Consider adding quoted text to emphasise specific design considerations and/or their impact

Reply:
For context, other blogs and articles use quotes as a form of call-out to emphasise specific sentences from the text

Reply:
Fixed in https://github.com/agrski/agrski.github.io/pull/40/commits/be4b17b10133bf35dac2b82ea592aaa83176e137